src/coreComponents/python/modules/geosx_mesh_doctor/parsing/generate_fractures_parsing.py

```diff
@@ -38,14 +39,14 @@ def fill_subparser(subparsers) -> None:
                    help=f"[bool]: Split policy if the fracture touches the boundary of the mesh. Defaults to true.")
     p.add_argument('--' + __FIELD_NAME,
                    type=str,
-                   help=f"[string]: If the \"{__FIELD_POLICY}\" {__POLICY} is selected, defines which field will be considered to define the fractures.")
+                   help=f"[string]: If the \"{__FIELD_POLICY}\" {__POLICY} is selected, defines which field will be considered to define the fractures. If the \"{__INTERNAL_SURFACES_POLICY}\" {__POLICY}  is selected, defines the name of the attribute will be considered to identify the fractures. ")
```

Comment:
Note that there also is a small doc here `src/docs/sphinx/pythonTools/mesh_doctor.rst` that you can use to add other comments if you think it's relevant.